int8 Id           # 0 is correct pinger, 1 is incorrect pinger
float64 Heading
float64 Magnitude
float64 TimeSince
